sample.tf

```diff
@@ -9,56 +9,161 @@ terraform {
 }
 
 provider "xray" {
-  //  supply ARTIFACTORY_USERNAME, _PASSWORD and _URL as env vars
+  //  supply ARTIFACTORY_URL and ARTIFACTORY_ACCESS_TOKEN as env vars
 }
-resource "xray_policy" "test" {
-  name        = "test-policy-name-severity"
-  description = "test policy description"
+
+resource "random_id" "randid" {
+  byte_length = 2
+}
+
+resource "xray_security_policy" "security1" {
```

Comment:
In this sample, I would like you to have an example of using the RT provider and this provider.

Reply:
will do in Xray Watch examples. When the user creates a policy, the Artifactory provider can't really add any value. It will be used in the Xray Watch tests. I'll create a scenario when the provider creates a repo, policy, and watch for that repo with the policy assigned to the watch.